From b847bb8c6ead622f2da19f405e56cab237c2f225 Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Mon, 18 May 2020 10:05:47 +0200
Subject: [PATCH] ppp: add sleep for unreliable DSL (PPP) connection

This patch is a hack. It should not be sent to upstream!
It was added in Turris OS 3.x without any further
explanation and it seems to help as a connection is established.

More details are included in:
https://gitlab.labs.nic.cz/turris/turris-build/issues/107

In branch of HBD there was removed PKG_RELEASE bump to avoid conflicts.
---
 package/network/services/ppp/files/ppp.sh | 1 +
 1 files changed, 1 insertions(+)

diff --git a/package/network/services/ppp/files/ppp.sh b/package/network/services/ppp/files/ppp.sh
index b553effd88..6c162c5f6b 100755
--- a/package/network/services/ppp/files/ppp.sh
+++ b/package/network/services/ppp/files/ppp.sh
@@ -133,6 +133,7 @@ ppp_generic_setup() {
 	[ "${keepalive_adaptive:-1}" -lt 1 ] && lcp_adaptive=""
 	[ -n "$connect" ] || json_get_var connect connect
 	[ -n "$disconnect" ] || json_get_var disconnect disconnect
+	sleep 10
 
 	proto_run_command "$config" /usr/sbin/pppd \
 		nodetach ipparam "$config" \
-- 
2.26.2

